x/user/types/events.go

```diff
@@ -2,23 +2,7 @@ package types
 
 // user module event types
 const (
-	EventTypePost             = "post"
```

Comment:
How come these are going away?

Reply:
copy pasta from curating module, not really used here.